src/Games/NexusMods.Games.AdvancedInstaller.UI/Pages/AdvancedInstallerPages/AdvancedInstallerPageViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Reactive.Disposables;
+using NexusMods.DataModel.Games;
+using NexusMods.DataModel.ModInstallers;
+using NexusMods.Paths;
+using NexusMods.Paths.FileTree;
+using ReactiveUI;
+
+namespace NexusMods.Games.AdvancedInstaller.UI;
+
+public class AdvancedInstallerPageViewModel : AViewModel<IAdvancedInstallerPageViewModel>,
+    IAdvancedInstallerPageViewModel
+{
+    /// <summary>
+    /// Constructor for the main Manual installer page, this currently contains the footer and body view models.
+    /// TODO: This isn't an actual Page yet, still needs conversion.
+    /// </summary>
+    /// <param name="modName">The display name for the mod to install.</param>
+    /// <param name="archiveFiles">A <see cref="FileTreeNode{RelativePath,ModSourceFileEntry}"/> of the files contained in the mod archive.</param>
+    /// <param name="register">The register containing the game locations.</param>
+    /// <param name="gameName">The display name of the game being managed.</param>
+    public AdvancedInstallerPageViewModel(string modName,
+        FileTreeNode<RelativePath, ModSourceFileEntry> archiveFiles,
+        GameLocationsRegister register,
+        string gameName)
+    {
+        BodyViewModel = new BodyViewModel(new DeploymentData(), modName, archiveFiles, register, null, gameName);
+        FooterViewModel = new FooterViewModel();
+        ShouldInstall = false;
+
+        FooterViewModel.InstallCommand = ReactiveCommand.Create(() => { ShouldInstall = true; },
+            this.WhenAnyValue(vm => vm.BodyViewModel.CanInstall));
+
+        this.WhenActivated(disposables =>
+        {
+            FooterViewModel.CancelCommand.Subscribe(_ => { ShouldInstall = false; }).DisposeWith(disposables);
+        });
```

Comment:
`FooterViewModel.CancelCommand` is marked as `[Reactive]` and has both a getter and a setter, which is not needed. It should be a get-only property.

`FooterViewModel.InstallCommand` should also be a get-only property. Creating and setting the command inside the `AdvancedInstallerPageViewModel` is not ideal. Instead, `FooterViewModel` should have a `CanInstall` property:

`[Reactive] public bool CanInstall { get; set; }`

The `AdvancedInstallerPageViewModel` will react to changes to `BodyViewModel.CanInstall` and forward the value to `FooterViewModel.CanInstall`.